src/model-helpers/DocumentList.js

```diff
@@ -0,0 +1,97 @@
+import {
+  getSolidDataset,
+  setThing,
+  getThingAll,
+  saveSolidDatasetAt,
+  createSolidDataset,
+  saveFileInContainer,
+  getThing,
+  removeThing,
+  deleteFile
+} from '@inrupt/solid-client';
+
+import { makeDocIntoThing, parseDocFromThing } from './Document';
+
+const parseFromDataset = async (docsDataset, session) => {
+  const docList = [];
+  const allDocThings = getThingAll(docsDataset);
+  await Promise.all(
+    allDocThings.map(async (userThing) => {
+      const docObject = await parseDocFromThing(userThing, session);
```

Comment:
parseDocFromThing() doesn't take in a session as a parameter.

Reply:
fixed